lib/cosmos/tools/tlm_grapher/tabbed_plots_tool/tabbed_plots_config.rb

```diff
@@ -428,7 +428,7 @@ def build_packet_to_data_objects_mapping
               # Build the index into the hash of the form "TARGET_NAME PACKET_NAME"
               # Note that + is used to create a new object and then << is used to concatenate
               # to the new object.
-              if packet_name.casecmp(Telemetry::LATEST_PACKET_NAME) == 0
+              if packet_name.casecmp(Telemetry::LATEST_PACKET_NAME).zero?
```

Comment:
Is .zero? better than == 0 in some way?

Reply:
No, it's just consistent across the codebase